optimize/util/optimize-commons/src/main/java/io/camunda/optimize/dto/optimize/query/entity/EntityNameResponseDto.java

```diff
@@ -19,5 +19,4 @@ public class EntityNameResponseDto {
   private String collectionName;
   private String dashboardName;
   private String reportName;
-  private String eventBasedProcessName;
```

Comment:
This is a separate breaking API change, has it been documented and handled by the FE?

Reply:
I am once again asking you to read the pr description